wemake_python_styleguide/violations/refactoring.py

```diff
@@ -37,6 +37,7 @@
    ImplicitSumViolation
    FalsyConstantCompareViolation
    WrongIsCompareViolation
+   WrongNamedKeywordViolation
```

Comment:
This is a best-practice. Please, move the violation.

Reply:
Done